tests/cases/elements/image.slint

```diff
@@ -13,6 +13,8 @@ TestCase := Rectangle {
         source-clip-x: 20;
     }
 
+    out property <image> with-border: @image-url("dog.jpg", 9slice(12 13 14 15));
```

Comment:
This tests basically the code generators, right?

Reply:
Yes. We do not have test for renderers other than the software renderer, and that's only tested with rust.